src/sections/Community/Handbook/repo-data.js

```diff
@@ -398,20 +398,56 @@ export const repo_data = [
   },
 
   {
-    category: "Installers Repos",
+    category: "Maintainers",
```

Comment:
Why do we have this twice?

Reply:
yes @debo19 actually I had created a separate list for maintainers in a  new file but since react accepts only 1 child hence i decided to add maintainers data too in the repo-data file.